runtimeconfig/manager_test.go

```diff
@@ -109,6 +215,36 @@ func TestNewOverridesManager(t *testing.T) {
 
 	// Make sure test limits were loaded.
 	require.NotNil(t, overridesManager.GetConfig())
+	conf := overridesManager.GetConfig().(*testOverrides)
+	require.Equal(t, 1234, conf.Overrides["user1"].Limit1)
+}
+
+func TestOverridesManagerMultipleFilesWithEmptyFile(t *testing.T) {
+	tempFiles, err := generateRuntimeFiles(t,
+		[]string{`overrides:
+  user1:
+    limit1: 100`,
+			``})
+	require.NoError(t, err)
+
+	// testing NewRuntimeConfigManager with overrides reload config set
```

Comment:
is this comment outdated? `NewRuntimeConfigManager` sounds like that's a function, but the name of the constructor is just `New()`, maybe it should be this instead?

```suggestion
	// testing runtime config manager with overrides reload config set
```